addons/l10n_ma/views/res_partner_views.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is it intended to have the the field appear 2 times on the partner page ? Because I see it already in the Accounting section, under the "Morocco" section.

![image](https://github.com/odoo/odoo/assets/115461192/d74aa681-7101-4a5f-a153-0d7405da250b)

Reply:
It shouldn't be the case. There is an enterprise PR. Have you seen it?

Reply:
I don't have the same behavior.

Reply:
My bad I indeed saw the enterprise PR but forgot to use it when testing :neutral_face:

Reply:
Guys, this field should have been there since 2016. Let's not waste anymore time please. Please merge.